### PR TITLE
testing how-to for k8s changes

### PR DESCRIPTION
**Pull request by shaneknapp**
i think that this will be quite useful.  :)

### CODE REVIEW

File: site/sitemap.xml [hunk 1]
```diff
@@ -760,19 +760,19 @@
   <changefreq>weekly</changefreq>
 </url>
 <url>
-  <loc>https://spark.apache.org/news/</loc>
+  <loc>https://spark.apache.org/streaming/</loc>
```
Comment (felixcheung) [inline site/sitemap.xml]: revert this file?

**Review by felixcheung — APPROVED** [review]
awesome, esp. the `behoove`

File: developer-tools.md [hunk 2]
```diff
@@ -175,6 +175,55 @@ You can check the coverage report visually by HTMLs under `/.../spark/python/tes
 
 Please check other available options via `python/run-tests[-with-coverage] --help`.
 
+<h4>Testing K8S</h4>
+
+If you have made changes to the K8S bindings in Apache Spark, it would behoove you to test locally before submitting a PR.  This is relatively simple to do, but it will require a local (to you) installation of [minikube](https://kubernetes.io/docs/setup/minikube/).  Due to how minikube interacts with the host system, please be sure to set things up as follows:
+
+-
```
Comment (tgravescs) [inline developer-tools.md]: extra empty line here?

**Comment by tgravescs** [comment]
this is great, thanks for writing these up Shane.

File: site/sitemap.xml (hunk 1 above)
Comment (shaneknapp) [reply site/sitemap.xml]: done

**Comment by shaneknapp** [comment]
> this is great, thanks for writing these up Shane.

not a problem!  after spending 2-3 days helping someone test their PR before finding out they were using the `--vm-driver=none` arg, i decided that this needed to be documented.

**Comment by tgravescs** [comment]
lgtm, note I didn't actually try the steps but probably will be in the next couple week.s

**Comment by shaneknapp** [comment]
> lgtm, note I didn't actually try the steps but probably will be in the next couple week.s

FWIW, these commands are pulled directly from the k8s integration test build, so things should be relatively simple to get set up on a dev's end.

regarding merging -- do i have perms to run the merge script in the website dir, or does someone else do that for me?  ;)

**Review by tgravescs — APPROVED** [review]

**Comment by tgravescs** [comment]
I'm not sure on permissions

**Comment by shaneknapp** [comment]
alright, i'll try and push from my end and see what happens.

**Comment by tgravescs** [comment]
ok let us know if it doesn't work

**Comment by shaneknapp** [comment]
woot!

**Pull request closed without merging**